migrations/1442160244719-edgesCreation.js

```diff
@@ -0,0 +1,44 @@
+var request = require('request-promise');
+
+var username = process.env.DB_USER;
+var password = process.env.DB_PASSWORD;
+var databaseName = process.env.DB_NAME;
+var databaseUrl = "http://localhost:2480/command/"+databaseName+"/sql";
+
+
+var command = function(str){
+  return request({
+    url: databaseUrl ,
+    method: "POST",
+    auth : {
+      'user' : username,
+      'pass' : password
+    },
+    body : str
+  })
+    .then(function (response) {
+      console.log("Command Sent Successfully");
+    })
+    .catch(function (response) {
+      console.log(response);
+    });
+}
+'use strict'
+
+exports.up = function(next) {
+	command('CREATE CLASS Follows EXTENDS E')
+  .then(() => command('CREATE CLASS AuthorOf EXTENDS E'))
+  .then(() => command('CREATE CLASS Favorites EXTENDS E'))
+  .then(() => command('CREATE CLASS ChildOf EXTENDS E'))
+  .then(() => command('CREATE CLASS Tagof EXTENDS E'))
```

Comment:
Missed this one. But now that I think about it, maybe `TaggedWith` is a better name... ?